packages/e2e/cypress/support/commands.js

```diff
@@ -19,4 +19,20 @@
 //
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+
+import { escapeRegExp } from 'lodash';
+Cypress.Commands.add('getInputField', (field) => {
+  const passwordFieldNames = ['Password', 'Confirm Password'];
+  const isPasswordField = passwordFieldNames.includes(field);
+  const regexString = `^${escapeRegExp(field)}$`;
+  const regex = new RegExp(regexString, 'i');
+
+  if (isPasswordField) {
+    // TODO: we should use cy.findByLabelText once our dom is cleaned up
+    return cy.findByPlaceholderText(regex);
+  } else {
+    return cy.findByRole('textbox', { name: regex });
+  }
+});
+
```

Comment:
This is the helper function that @slaymance and I landed on. Only engineering problem I see is that we are hard coding which fieldnames are passwords (`passwordFieldNames`). We also considered having a more explicit option to set passwords, but we still needed to distinguish password fields on the `integration/common/index.tsx` file.  This implementation is fine as of now, I'm just thinking that this might need to be updated if we test for customized placeholders which then wouldn't have `['Password', 'Confirm Password']` names. 

I'm okay to have this function determine password fields implicitly for now, then we can revisit this later when it's needed, instead of abstracting early.

Reply:
Good summary, and I think once we get our primitives across all frameworks (namely `PasswordField`) then we can lean on the `Password` and `Confirm Password` labels that come from `I18n` to query for these fields.